README.rst

```diff
@@ -36,11 +36,16 @@ Usage
     # ...and into which we install our project and its testing requirements.
     $ env/bin/pip install -e ".[testing]"
 
-3. Configure the database.
+3. Initialize the database.
 
 .. code-block:: bash
 
-    $ env/bin/initialize_myproj_db development.ini
+    # Use alembic to generate revisions before you run initialize.
+    $ env/bin/alembic -c development.ini revision --autogenerate -m "init"
+    # Then you can use alembic to upgrade to that revision.
+    $ env/bin/alembic -c development.ini upgrade head
+    # Or initialize the database.
+    $ env/bin/initialize_{{ cookiecutter.repo_name }_db development.ini
```

Comment:
I have a sinking feeling that enough users will just run through these steps in order, copy-pasting without first thinking or reading the entire stanza. It would be helpful to use formatting and language to inform the user. Here are suggestions:

```
3. Initialize the database. Read all the options first, then choose according to your needs.
```

```
    # Use alembic to generate revisions before you run initialize.
    $ env/bin/alembic -c development.ini revision --autogenerate -m "init"
    # Then you can use alembic either to (1) upgrade to that revision...
    $ env/bin/alembic -c development.ini upgrade head
    # ...or (2) initialize the database.
    $ env/bin/initialize_{{ cookiecutter.repo_name }}_db development.ini
```

Note there was a missing trailing `}`. I like this abstraction, because I recall a couple of times where beginners copy-pasted `env/bin/initialize_myproj_db development.ini` and wondered why it didn't work. It stands out with this change.